Makefile

```diff
@@ -43,12 +43,9 @@ functions-build:
 check-headers-file:
 	scripts/check-headers-file.sh
 
-production-build: module-check ## Build the production site and ensure that noindex headers aren't added
+production-build: module-check ## Build the production or a release-branch site and ensure that noindex headers aren't added
```

Comment:
Hi @sftim, you are correct that the comment isn't accurate, and thanks for pointing that out. I can fix that soon, but I think that we need to address a broader issue.

With the hope of us finding a mutually agreeable solution to #36333, would you be open to considering that we distinguish build kinds as follows:

- `main-production-build`: an indexed, minified, asset-fingerprinted site deploy
- `deprecated-production-build` (what is currently called a "release-branch"):
    an **non-indexed**, minified, asset-fingerprinted site deploy
- `build` - a non-production build, hence non-indexed, no minification or asset fingerprinting.

Reply:
@chalin if possible, try to have a discussion about the overall approach in the [issue](https://github.com/kubernetes/website/issues/36333) rather than in a PR.

Reply:
Yes, moved the discussion to https://github.com/kubernetes/website/issues/36333#issuecomment-1294012582. Hopefully that provides a clearer picture.